VideoLocker/src/main/java/org/edx/mobile/model/api/CourseEntry.java

```diff
@@ -248,22 +252,72 @@ public boolean isGroupAvailable(SocialUtils.SocialType type) {
 
     }
 
-    public String getDescription(Context context){
+    @SuppressLint("SimpleDateFormat")
```

Comment:
Hmm, we're getting the warning because we're using a custom non-localized date format (`MMMM dd`). Maybe we should consider localizing it? @cc @aleffert @bguertin